lib/ex_webrtc/rtp_transceiver.ex

```diff
@@ -122,27 +122,30 @@ defmodule ExWebRTC.RTPTransceiver do
   @doc false
   @spec to_answer_mline(t(), ExSDP.Media.t(), Keyword.t()) :: ExSDP.Media.t()
   def to_answer_mline(transceiver, mline, opts) do
-    offered_direction = SDPUtils.get_media_direction(mline)
-    direction = get_direction(offered_direction, transceiver.direction)
-    opts = Keyword.put(opts, :direction, direction)
-
     # Reject mline. See RFC 8829 sec. 5.3.1 and RFC 3264 sec. 6.
     # We could reject earlier (as RFC suggests) but we generate
     # answer mline at first to have consistent fingerprint, ice_ufrag and
     # ice_pwd values across mlines.
+    # We also set direction to inactive, even though JSEP doesn't require it.
+    # See see https://github.com/w3c/webrtc-pc/issues/2927
     cond do
       transceiver.codecs == [] ->
         # there has to be at least one format so take it from the offer
         codecs = SDPUtils.get_rtp_codec_parameters(mline)
         transceiver = %__MODULE__{transceiver | codecs: codecs}
+        opts = Keyword.put(opts, :direction, :inactive)
         mline = to_mline(transceiver, opts)
         %ExSDP.Media{mline | port: 0}
 
       transceiver.stopping == true or transceiver.stopped == true ->
+        opts = Keyword.put(opts, :direction, :inactive)
         mline = to_mline(transceiver, opts)
         %ExSDP.Media{mline | port: 0}
 
       true ->
+        offered_direction = SDPUtils.get_media_direction(mline)
+        direction = get_direction(offered_direction, transceiver.direction)
+        opts = Keyword.put(opts, :direction, direction)
```

Comment:
Was this previously unhandled? How did we handle the `sRD(offer)` -> `tr.direction=inactive` -> `sLD(answer)` scenario then?